nebula-logger/core/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -89,14 +89,55 @@ private class LogEntryEventHandler_Tests {
         System.assertEquals(0, countOfLogEntries);
     }
 
+    @IsTest
+    static void it_should_not_create_log_or_log_entry_data_when_platform_event_storage_location_is_null_in_logger_scenario_rule() {
```

Comment:
what's the use-case for not having a storage location on the scenario rule? wouldn't that be a waste of CPU time to even create a log entry record in memory at all?

Reply:
Several orgs use it for publishing platform events (and they have external systems subscribing to the events) without using any data storage in the org. It's a bit of an edge case, but it seems like bigger orgs with a lot of logging will sometimes disable storage on platform for certain users, and have the platform events stored off-platform instead.

Reply:
Right, right - ok, that makes sense!